ui/app/AppLayouts/Profile/views/AppearanceView.qml

```diff
@@ -19,22 +18,17 @@ import "../stores"
 SettingsContentBase {
     id: appearanceView
 
-    property AppearanceStore appearanceStore
+    property int currentFontSize: 2
+    signal fontSizeChanged(value: int)
```

Comment:
This is not valid syntax; should be `signal fontSizeChanged(int value)`

Actually surprised it works at all